docs(ai): smart labeling - MTA-1571 (#210)
scaleway/docs-content: ai/smart-labeling/quickstart.mdx

---
title: Smart Labeling Quickstart
description: This page shows you how to get started with Scaleway Smart Labeling.
dates:
  validation: 2021-11-04
  posted: 2021-11-04
---

> Scaleway [Smart Labeling](ADD_LINK) is a platform that allows you to annotate images or videos (whether by yourself or via a team of annotators). The annotation software that Smart Labeling is currently using is CVAT: the [open-source Computer Vision Annotation Tool](https://github.com/openvinotoolkit/cvat) developed by Intel. Smart Labeling facilitates using CVAT in the cloud-native manner, with labeling software running on-demand, annotations safely persisted in an automatically backed up database, and datasets stored in cost-efficient S3 buckets.

Reviewer: il manque le lien

Scaleway Smart Labeling comes with everything needed to start annotating: whether on your own or with the help of others (referred to as annotators). Annotators do not need to create Scaleway accounts, they can log in directly to the CVAT annotation interface, whose login page is publicly accessible.

<Message type="requirement">

  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com) 
  - You have [created an Object Storage bucket](/storage/object/quickstart/)

</Message>

<Message type="note">

Currently, Smart Labeling only supports the annotation of image and video file formats. If you have other files stored in the linked buckets, you will not be able to view them in the annotation interface, but their presence will not impact the application.

</Message>

## How to create a Smart Labeling workspace

1. Click **Smart Labeling** in the **AI** section of the side menu. The Smart Labeling splash screen displays:

  <Lightbox src="scaleway-smart-labeling-splash.png" alt="" />

2. Click **Create a labeling workspace**. The workspace creation wizard displays.

> 3. Enter the required information for your labeling worksapce:

Reviewer: ```suggestion
3. Enter the required information for your labeling workspace:
```

    * Enter a Name and an optional description for the workspace.
    * Select the Object Storage bucket(s) you want to use with your workspace.
    * Enter a username for the admin user.
    * Enter the email address of the admin user.
    * Enter a secret password for the admin user.

4. Confirm by clicking **Create a labeling workspace**. Your workspace will be created.

Once ready, the workspace displays in the list of your Smart Labeling workspaces.

## How to access your workspace

Click **Smart Labeling** in the **AI** section of the side menu. The list of your workspaces displays:

  <Lightbox src="scaleway-smart-labeling-list.png" alt="" />

The information of your workspace displays:

  <Lightbox src="scaleway-smart-labeling-worksapce.png" alt="" />

  <Message type="tip">

  Note that what CVAT refers to as the **superuser**, who has full access to the administration panel of the annotation server, is the same person as the admin user of the labeling workspace. You can use these credentials to login.

  </Message>

### How to access the annotation server

> From your Smart Labeling workspace, click **Go to my annotation server** in the workspace overview. The CVAT interface dispalys. 

Reviewer: I think can be numbered as the first step (and start with an action verb).

</Message>

1. Enter the credentials for the admin user of the Smart Labeling workspace and click **Login**. The CVAT dashboard displays:

  <Lightbox src="scaleway-smart-labeling-dashboard.png" alt="" />

2. Create Tasks to annotate datasets of your choice with [image-level tags, as well as points, bounding boxes, polygons, and 3D boxes](https://openvinotoolkit.github.io/cvat/docs/manual/basics/types-of-shapes/) and assign them to your users.

3. Users can login into the CVAT interface to start annotating images in their tasks.

  <Message type="important">

  Annotators can create users and login to the annotation server directly from this interface. 

  </Message>

### How to assign tasks to annotators

Although each annotator can immediately use the account to log into the annotation server, they will not have access to the annotation tasks you have created, unless you assign the task to them, or you change their permissions by going into the administration panel of the CVAT server.

> 1. Click on your username in the upper right corner and click **Admin page** in the drop-down menu.

Reviewer: ```suggestion
1. Click your username in the upper right corner and click **Admin page** in the drop-down menu.
```

---
title: Smart Labeling Quickstart

> description: This page shows you how to get started with Scaleway Elements Bare Metal Servers.

Reviewer 2: ```suggestion
description: This page shows you how to get started with Scaleway Smart Labeling.
```

dates:
  validation: 2021-11-04
  posted: 2021-11-04
---

> Scaleway [Smart Labeling)(ADD_LINK) is a plattform that allows you to annnotate images or videos (whether by yourself or via a team of annotators). The annotation interface that Smart Labeling is currently using is CVAT: the [open-source Computer Vision Annotation Tool](https://github.com/openvinotoolkit/cvat) developed by Intel. Smart Labeling facilitates using CVAT in the cloud-native manner, with labeling software running on-demand, annotations safely persisted in an automatically backed up database, and datasets stored in cost-efficient S3 buckets.

Reviewer: ```suggestion
Scaleway [Smart Labeling](ADD_LINK) is a platform that allows you to annotate images or videos (whether by yourself or via a team of annotators). The annotation interface that Smart Labeling is currently using is CVAT: the [open-source Computer Vision Annotation Tool](https://github.com/openvinotoolkit/cvat) developed by Intel. Smart Labeling facilitates using CVAT in the cloud-native manner, with labeling software running on-demand, annotations safely persisted in an automatically backed up database, and datasets stored in cost-efficient S3 buckets.
```

> Scaleway [Smart Labeling)(ADD_LINK) is a plattform that allows you to annnotate images or videos (whether by yourself or via a team of annotators). The annotation interface that Smart Labeling is currently using is CVAT: the [open-source Computer Vision Annotation Tool](https://github.com/openvinotoolkit/cvat) developed by Intel. Smart Labeling facilitates using CVAT in the cloud-native manner, with labeling software running on-demand, annotations safely persisted in an automatically backed up database, and datasets stored in cost-efficient S3 buckets.

Reviewer 2: ```suggestion
Scaleway [Smart Labeling](ADD_LINK) is a platform that allows you to annotate images or videos, either by yourself or via a team of annotators. Smart Labeling currently uses Intel's CVAT as its interface: [open-source Computer Vision Annotation Tool](https://github.com/openvinotoolkit/cvat). Smart Labeling facilitates the use of CVAT in a cloud-native manner, with labeling software running on-demand, annotations safely persisting in an automatically backed-up database, and datasets stored in cost-efficient S3 buckets.
```

> Our solution comes with everything needed to start annotating: whether on your own or with the help of others (referred to as annotators). Annotators do not need to create Scaleway accounts, thy can log in directly to the CVAT annotation interface, whose login page is publicly accessible.

Reviewer: ```suggestion
Scaleway Smart Labeling comes with everything needed to start annotating: whether on your own or with the help of others (referred to as annotators). Annotators do not need to create Scaleway accounts, thy can log in directly to the CVAT annotation interface, whose login page is publicly accessible.
```

> Our solution comes with everything needed to start annotating: whether on your own or with the help of others (referred to as annotators). Annotators do not need to create Scaleway accounts, thy can log in directly to the CVAT annotation interface, whose login page is publicly accessible.

Reviewer 2: ```suggestion
Our solution comes with everything you need to start annotating. Only the designated Admin needs to create a Scaleway accout - annotators can simply log in directly to the publicly-accessible CVAT annotation interface without needing their own Scaleway accounts.
```

> 

Reviewer 2: For rest, apply suggestions from the how tos to the quick start bits copied/pasted too

  - You have [created an Object Storage bucket](/storage/object/quickstart/)

</Message>

<Message type="note">

Currently, Smart Labeling only supports the annotation of image and video file formats. If you have other files stored in the linked buckets, you will not be able to view them in the annotation interface, but their presence will not impact the application.

</Message>

## How to create a Smart Labeling workspace

1. Click **Smart Labeling** in the **AI** section of the side menu. The Smart Labeling splash screen displays:

<Lightbox src="scaleway-smart-labeling-splash.png" />

2. Click **Create a labeling workspace**. The workspace creation wizard displays.

3. Enter the required information for your labeling worksapce:
    * Enter a Name and a description for the workspace.
    * Select the Object Storage bucket you want to use with your workspace.
    * Enter a username for the admin user.
    * Enter the email address of the admin user. 
    * Enter a secret password for the admin user.

4. Confirm by clicking **Create a labeling workspace**. Your workspace will be created.

> Once ready the workspace displaces in the list of your Smart Labeling workspaces.

Reviewer: ```suggestion
Once ready, the workspace displays in the list of your Smart Labeling workspaces.
```

## How to access your workspace

1. Click **Smart Labeling** in the **AI** section of the side menu. The list of your workspaces displays:

<Lightbox src="scaleway-smart-labeling-list.png" />

> 2. The information of your workspace displays:

Reviewer: this is not a step. I suggest that all steps get removed in this case

</Message>

### How to access the annotation server

From your Smart Labeling workspace, click **Go to my annotation server** in the workspace overview. The CVAT interface dispalys. 

<Message type="note">

Alternatively you can use the **Annotation server URL** to access the server. You can find it in the workspace information block. This URL will take you (or the annotator) to the CVAT annotation interface. 

</Message>

1. Enter the credentials for the admin user of the Smart Labeling workspace and click **Login**. The CVAT dashboard displays:

<Lightbox src="scaleway-smart-labeling-dashboard.png" />

2. Ceate Tasks to annotate datasets of your choice with [image-level tags, as well as points, bounding boxes, polygons, and 3D boxes](https://openvinotoolkit.github.io/cvat/docs/manual/basics/types-of-shapes/) and assign them to your users.

3. Users can login into the CVAT interface to start annotating images in their tasks.

<Message type="important">

> Annotatotors can create users and login to the annotation server directly from this interface. 

Reviewer: ```suggestion
Annotators can create users and login to the annotation server directly from this interface. 
```

</Message>

### How to assign tasks to annotators

Although each annotator can immediately use the account to log into the annotation server, they will not have access to the annotation tasks you have created, unless you assign the task to them, or you change their permissions by going into the administration panel of the CVAT server.

1. Click on your username in the upper right corner and click **Admin page** in the drop-down menu.

> 2. Select the tasks and roles you want to add to each user and validate (TO ADD MORE CONTENT HERE)

Reviewer: just reminding you that there are parenthesis are here in case you forgot
